src/graphql/Dashboard/heartbeat.gql

```diff
@@ -1,10 +1,10 @@
-query Heartbeat($projectId: uuid, $timestamp: timestamptz, $state: String) {
+query Heartbeat($projectId: uuid, $timestamp: timestamptz, $state: String, $filterOutState: String) {
   flow_run(
     limit: 10
     where: {
       flow: { project_id: { _eq: $projectId } }
       state_timestamp: { _gte: $timestamp }
-      state: { _eq: $state }
+      state: { _eq: $state, _neq: $filterOutState }
```

Comment:
This feels a little convoluted and will be confusing when separated from the motivation in this PR, because if someone weren't aware of how Hasura treats `null` variables, this looks like we're saying "give me flow runs that match a state but not a different state", which doesn't make much sense.

However, I recognize that it fits our usage. My suggestion would be to make it a `_nin` clause instead of a `_neq` clause, to make it easier to filter out potentially other states going forward. 
```suggestion
      state: { _eq: $state, _nin: $filterOutStates }
```

Reply:
@jlowin this makes sense but should $state also be made a list whilst i'm making this change?

Reply:
I'm more hesitant to touch things that are already in use because it may require more updates elsewhere